### PR TITLE
chore: run tests with full-icu

### PR DESCRIPTION
**Pull request by montezume**
#### Summary

We want to be able to test formatting in different languages. Node doesn't support this out of the box. So we use `full-icu`. But `full-icu` is broken with yarn. So we bring the dependency internal to a "vendors" folder until they fix the problem.

cc @islam3zzat

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -52,8 +52,9 @@
     "format:js": "prettier --write '**/*.js'",
     "format:md": "prettier --write --parser markdown '**/*.md'",
     "format:yaml": "prettier --write --parser yaml '**/*.yaml'",
-    "test": "jest --config jest.test.config.js",
-    "test:watch": "jest --config jest.test.config.js --watch",
+    "postinstall": "cd vendors && npm install",
```
Comment (tdeekens) [inline package.json]: Nit: `yarn --cwd` or `npm --prefix` should work too instead of `cd.

File: .circleci/config.yml [hunk 2]
```diff
@@ -52,7 +52,7 @@ commands:
   unit_test:
     description: 'Running unit tests'
     steps:
-      - run: yarn cross-env NODE_ICU_DATA=./vendors/node_modules/full-icu run jest --projects jest.test.config.js --maxWorkers=3 --reporters jest-silent-reporter
+      - run: yarn run cross-env NODE_ICU_DATA=vendors/node_modules/full-icu jest --projects jest.test.config.js --maxWorkers=3 --reporters jest-silent-reporter
```
Comment (tdeekens) [inline .circleci/config.yml]: Nit: the run scripts use `npm` CI uses `yarn`? Maybe settle on one CLI?

Comment (montezume) [reply .circleci/config.yml]: which run script do you mean?

Comment (montezume) [reply .circleci/config.yml]: There is currently an issue with installing `full-icu` with `yarn`. See [issue](https://github.com/unicode-org/full-icu-npm/issues/9).

Comment (tdeekens) [reply .circleci/config.yml]: Ah sorry :)

**Pull request merged**